src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -343,6 +365,16 @@ MUST be present **in this specific order**:
 | z               | REQUIRED              | Z position. If electrodes are in 2D space this should be a column of `n/a` values.                                               |
 | size            | REQUIRED              | Surface area of the electrode, units MUST be in `mm^2`.                                                                          |
 
+{{ MACROS___make_columns_table(
+   {
+      "name_electrodes": "REQUIRED",
+      "x": "REQUIRED",
```

Comment:
This previously contained the following information:

> The positions of the center of each electrode in xyz space. Units are specified in space-<label>_coordsystem.json.

I think that should be preserved. Possibly in the spec text above the table.